src/client/Client.cc

```diff
@@ -12579,7 +12579,13 @@ int Client::_statfs(Inode *in, struct statvfs *stbuf,
   stbuf->f_files = total_files_on_fs;
   stbuf->f_ffree = -1;
   stbuf->f_favail = -1;
-  stbuf->f_fsid = -1;       // ??
```

Comment:
So outside a sub-volume f_fsid will be -1? I guess that will work.

I do hope using an inode number is sufficiently unique if multiple filesystems are involved.